pkgs/by-name/di/dircolors-solarized/package.nix

```
@@ -0,0 +1,38 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  unstableGitUpdater,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dircolors-solarized";
+  version = "52bfa164e4388ee232f6a9235f62e8482e1f1767";
+
+  src = fetchFromGitHub {
+    owner = "seebi";
+    repo = "dircolors-solarized";
+    rev = finalAttrs.version;
+    hash = "sha256-+2t9OsyD9QkamsFbgmgehBrfszBQmv1Y0C94T4g16GI=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    for f in dircolors.*; do
+      install -Dm644 $f $out/''${f#dircolors.}
+    done
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = unstableGitUpdater { };
+  meta = {
+    description = "Repository of themes for GNU ls (configured via GNU dircolors) that support Ethan Schoonover’s Solarized color scheme";
```

This is way too long. You can add it to `longDescription`, for description:

```suggestion
    description = "Repository of themes for GNU, supporting Solarized color scheme";
```